core/src/test/scala/unit/kafka/server/ReplicaFetcherThreadTest.scala

```diff
@@ -122,7 +122,7 @@ class ReplicaFetcherThreadTest {
 
 
     //Expectations
-    expect(logManager.truncateTo(anyObject())).once
+    expect(logManager.truncateTo(anyObject())).anyTimes()
```

Comment:
Hmm, do you know why the truncation needs to be called more than once now?

Reply:
Sure. This is because two partitions, t1p0 and t1p1, are added to this ReplicaFetcherThread. And now ReplicaFetcherThread.maybeTruncate() can possibly call truncateTo() once per partition.

Reply:
Maybe it should verify that it's only called twice then?

Reply:
@ijuma Sure. That has been done in the patch.